README.rst

```diff
@@ -214,8 +214,8 @@ Choose from the list of available rules:
 
   Configuration options:
 
-  - ``syntax`` (``'long'``, ``'short'``): whether to use the ``long`` or ``short`` array
-    syntax; defaults to ``'long'``
+  - ``syntax`` (``'long'``, ``'short'``): whether to use the ``long`` or ``short`` array syntax;
```

Comment:
IMO we should have a concise README (Markdown) and move all details here to doc/ (reStructuredText), but that's out of the scope of this PR.